Wangscape/noise/module/Wrapper.cpp

```diff
@@ -0,0 +1,67 @@
+#include "Wrapper.h"
+
+namespace noise
+{
+namespace module
+{
+
+template<> void Wrapper<ModuleGroup>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Billow>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Perlin>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<RidgedMulti>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Turbulence>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Voronoi>::setSeed(int seed)
+{
+    module.SetSeed(seed);
+}
+
+template<> void Wrapper<Displace>::setDisplaceModule(int index, const Module& source_module)
+{
+    switch (index)
+    {
+    case 0:
+        module.SetXDisplaceModule(source_module);
+        break;
+    case 1:
+        module.SetXDisplaceModule(source_module);
+        break;
+    case 2:
+        module.SetXDisplaceModule(source_module);
+        break;
+    default:
+        throw std::runtime_error("Tried to set displace module with index greater than 2");
```

Comment:
Because two of those calls should be to `SetYDisplaceModule` and `SetZDisplaceModule`.